.rultor.yml

```diff
@@ -0,0 +1,13 @@
+docker:
+  image: yegor256/rultor-image:1.13.0
+architect:
+  - yegor256
+assets:
```

Comment:
@c71n93 `assets` are not needed in this script, you can remove this section

Reply:
> @c71n93 `assets` are not needed in this script, you can remove this section

Fixed it